www/assets/css/home.scss

```diff
@@ -381,7 +381,7 @@ $searchbox-height: 50px;
     width: 100%;
     background-size: cover !important;
     display: flex;
-    align-items: end;
+    align-items: flex-end;
```

Comment:
We were getting warnings about this issue previously. (`"end" has mixed support... use "flex-end" instead.`)

The new version of dev server cares more (won't load page). Hence fixing it here.